features/FEATURE_BLE/targets/TARGET_NORDIC/TARGET_NRF5/source/nRF5xGattServer.cpp

```diff
@@ -179,6 +189,11 @@ ble_error_t nRF5xGattServer::addService(GattService &service)
                 nrfCharacteristicHandles[characteristicCount].user_desc_handle
             );
         }
+        if (p_presentation_format_descriptor) {
+            p_presentation_format_descriptor->setHandle(
+                nrfCharacteristicHandles[characteristicCount].value_handle
```

Comment:
Unfortunately the Softdevice doesn't return the handle of the presentation format descriptor. I would set this handle to `GattAttribute::INVALID_HANDLE`.